examples/cn-cma-babj.nmic.prediction-forecast.json

```diff
@@ -1,5 +1,6 @@
 {
-    "id": "urn:wmo:md:cn-cma-babj:data.core.weather.prediction.forecast.shortrange.probabilistic.global",
+    "amComment": "short-range or medium-range? global or limited-area? are all of the discipline topics relevant? Is there a real link for data access?",

```

Comment:
@Amienshxq I have a couple consistency questions:
1. short-range or medium-range? 
2. global or limited-area? 
3. are all of the discipline topics relevant? 
4. Is there a real link for data access?"